src/scales/scale.time.js

```diff
@@ -175,8 +175,25 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var majorUnitStart = startTick;
+			if (options.majorUnit) {
+				majorUnitStart = moment(startTick).add(1, options.majorUnit).startOf(options.majorUnit);
+			}
+			var startRange = majorUnitStart.valueOf() - startTick;
+			var startFraction = startRange % (interval[options.unit].size * stepSize);
+			var alignedTick = startTick;
+			ticks.push(startTick);
+			if (startTick !== alignedTick + startFraction &&
+				options.majorUnit &&
```

Comment:
Ok, well if it makes ESLint fail, then nevermind. @simonbrunel do you have any convention about whether the `&&` should be at the end of the line or the beginning of the next line?